src/Dialog/Dialog.js

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react';
+import { findDOMNode } from 'react-dom';
+import classNames from 'classnames';
+
+class Dialog extends React.Component {
+    static propTypes = {
+        className: PropTypes.string,
+        isOpen: PropTypes.bool
```

Comment:
Most of the prop we use in react-mdl doesn't use the prefix `is`. Do you think it would be better to only use `open`?
The native dialog also use an `open` attribute.